src/main/java/team/themoment/hellogsmv3/domain/application/service/ModifyApplicationService.java

```diff
@@ -0,0 +1,269 @@
+    }
+
+    private GraduateMiddleSchoolGrade createGraduateMiddleSchoolGrade(AbstractApplication application) {
+        return GraduateMiddleSchoolGrade.builder()
+                // 환산 로직은 추후 구현 예정
+                .id(application.getMiddleSchoolGrade().getId())
+                .percentileRank(BigDecimal.ONE)
+                .attendanceScore(BigDecimal.ONE)
+                .volunteerScore(BigDecimal.ONE)
+                .build();
+    }
+
+    private GraduateApplication createGraduateApplication(
+            ApplicationReqDto dto,
+            GraduatePersonalInformation graduatePersonalInformation,
+            GraduateMiddleSchoolGrade graduateMiddleSchoolGrade,
+            Applicant applicant,
+            AbstractApplication application
+    ) {
+        return GraduateApplication.builder()
+                .id(application.getId())
+                .personalInformation(graduatePersonalInformation)
+                .middleSchoolGrade(graduateMiddleSchoolGrade)
+                .statusParameter(AbstractApplicationStatusParameter.builder()
+                        .finalSubmitted(application.getFinalSubmitted())
+                        .printsArrived(application.getPrintsArrived())
+                        .subjectEvaluationResult(application.getSubjectEvaluationResult())
+                        .competencyEvaluationResult(application.getCompetencyEvaluationResult())
+                        .registrationNumber(application.getRegistrationNumber())
+                        .desiredMajors(DesiredMajors.builder()
+                                .firstDesiredMajor(dto.firstDesiredMajor())
+                                .secondDesiredMajor(dto.secondDesiredMajor())
+                                .thirdDesiredMajor(dto.thirdDesiredMajor())
+                                .build())
+                        .finalMajor(application.getFinalMajor())
+                        .build())
+                .applicant(applicant)
+                .build();
+    }
+
+    private GedPersonalInformation createGedPersonalInformation(ApplicationReqDto dto, AbstractApplication application) {
+        return GedPersonalInformation.builder()
+                .id(application.getPersonalInformation().getId())
+                .superParam(AbstractPersonalInformationParameter.builder()
+                        .address(dto.address())
+                        .applicantImageUri(dto.applicantImageUri())
+                        .detailAddress(dto.detailAddress())
+                        .graduation(dto.graduation())
+                        .phoneNumber(dto.telephone())
+                        .guardianName(dto.guardianName())
+                        .relationWithApplicant(dto.relationWithApplicant())
+                        .build())
+                .build();
+    }
+
+    private GedMiddleSchoolGrade createGedMiddleSchoolGrade(AbstractApplication application) {
+        return GedMiddleSchoolGrade.builder()
+                // 환산 로직은 추후 구현 예정
```

Comment:
이런건 TODO 로 마킹하면 IDE inspection 에도 잡힐 것 같습니다.

```suggestion
                // TODO 환산 로직은 추후 구현 예정
```

Reply:
3dcd3c46c0ecb62cf68ca35d4e204d05ac68e604

넵 TODO 키워드를 추가하였습니다.